Zig CI is broken

```
inversion.zig:49:71: error: root struct of file 'meta' has no member named 'bitCount'
            fiat.selectznz(&v, @truncate(u1, f[f.len - 1] >> (std.meta.bitCount(Field.Word) - 1)), v, v_opp);
                                                              ~~~~~~~~^~~~~~~~~
referenced by:
    test.Field inversion: inversion.zig:100:19
    remaining reference traces hidden; use '-freference-trace' to see all reference traces
```
@jedisct1 how should we fix this?

--- a/fiat-zig/src/main.zig
+++ b/fiat-zig/src/main.zig
@@ -5,11 +5,11 @@ fn testVector(comptime fiat: type, expected_s: []const u8) !void {
     // Find the type of the limbs and the size of the serialized representation.
     const repr = switch (@typeInfo(@TypeOf(fiat.fromBytes))) {
         .Fn => |f| .{
-            .Limbs = switch (@typeInfo(f.args[0].arg_type.?)) {
+            .Limbs = switch (@typeInfo(f.params[0].type.?)) {
                 .Pointer => |p| p.child,
                 else => unreachable,
             },
-            .Bytes = f.args[1].arg_type.?,
+            .Bytes = f.params[1].type.?,
         },
         else => unreachable,
     };

--- a/inversion/zig/inversion.zig
+++ b/inversion/zig/inversion.zig
@@ -46,7 +46,7 @@ fn fieldElement(comptime Field: type) type {
             }
             var v_opp: Field.Limbs = undefined;
             fiat.opp(&v_opp, v);
-            fiat.selectznz(&v, @truncate(u1, f[f.len - 1] >> (std.meta.bitCount(Field.Word) - 1)), v, v_opp);
+            fiat.selectznz(&v, @truncate(u1, f[f.len - 1] >> (@bitSizeOf(Field.Word) - 1)), v, v_opp);
 
             const precomp = comptime x: {
                 var precomp: Field.Limbs = undefined;

